src/Commands/ToggleDone.ts

```diff
@@ -66,27 +73,27 @@ const toggleLine = ({ line, path }: { line: string; path: string }): string => {
         // The task regex will match checklist items.
         const regexMatch = line.match(Task.taskRegex);
         if (regexMatch !== null) {
-            toggledLine = toggleChecklistItem({ regexMatch });
+            // Toggle the status of the checklist item.
```

Comment:
What was the benefit of removing the function `toggleChecklistItem()`?

And was there a change in behaviour?